src/Utilities/Phone_number.json

```diff
@@ -2,1588 +2,2447 @@
       "format_regex": "^(\\d{4})(\\d{3})(\\d{4})$"
     },
     {
       "country_code": "US",
+      "country_name": "United States",
+      "country_flag": "🇺🇸",
       "phone_number_code": "+1",
       "validation_regex": "^(\\+?1)?[2-9]\\d{2}[2-9](?!11)\\d{6}$",
       "format_example": "(123) 456-7890",
       "format_regex": "^\\(\\d{3}\\) \\d{3}-\\d{4}$"
     },
     {
       "country_code": "UY",
+      "country_name": "Uruguay",
+      "country_flag": "🇺🇾",
       "phone_number_code": "+598",
       "validation_regex": "^(\\+?598|0)9\\d{7}$",
       "format_example": "091234567",
       "format_regex": "^(\\d{2})(\\d{3})(\\d{4})$"
     },
     {
       "country_code": "UZ",
+      "country_name": "Uzbekistan",
+      "country_flag": "🇺🇿",
       "phone_number_code": "+998",
       "validation_regex": "^(\\+?998\\-?)[13-79]\\d{7}$",
       "format_example": "901234567",
       "format_regex": "^(\\d{3})(\\d{6})$"
     },
     {
       "country_code": "VU",
+      "country_name": "Vanuatu",
+      "country_flag": "🇻🇺",
       "phone_number_code": "+678",
       "validation_regex": "^(\\+?678)\\d{5}$",
       "format_example": "59123",
       "format_regex": "^\\d{5}$"
     },
     {
       "country_code": "VE",
+      "country_name": "Venezuela",
+      "country_flag": "🇻🇪",
       "phone_number_code": "+58",
       "validation_regex": "^(\\+?58)?[41269]\\d{7}$",
       "format_example": "04121234567",
       "format_regex": "^(\\d{4})(\\d{7})$"
     },
     {
       "country_code": "VN",
+      "country_name": "Vietnam",
+      "country_flag": "🇻🇳",
       "phone_number_code": "+84",
       "validation_regex": "^(\\+?84|0)1\\d{9}$",
       "format_example": "0912345678",
       "format_regex": "^(\\d{3})(\\d{3})(\\d{4})$"
     },
     {
       "country_code": "WF",
+      "country_name": "Wallis and Futuna",
+      "country_flag": "🇼🇫",
       "phone_number_code": "+681",
       "validation_regex": "^(\\+?681)\\d{6}$",
       "format_example": "501234",
       "format_regex": "^\\d{6}$"
     },
     {
       "country_code": "EH",
+      "country_name": "Western Sahara",
+      "country_flag": "🇪🇭",
       "phone_number_code": "+212",
       "validation_regex": "^(\\+?212)\\d{9}$",
       "format_example": "0651234567",
       "format_regex": "^(\\d{2})(\\d{3})(\\d{2})(\\d{2})$"
     },
     {
       "country_code": "YE",
+      "country_name": "Yemen",
+      "country_flag": "🇾🇪",
       "phone_number_code": "+967",
       "validation_regex": "^(\\+?967)?[12569]\\d{7}$",
       "format_example": "712345678",
       "format_regex": "^(\\d{3})(\\d{6})$"
     },
     {
       "country_code": "ZM",
+      "country_name": "Zambia",
+      "country_flag": "🇿🇲",
       "phone_number_code": "+260",
       "validation_regex": "^(\\+?260)\\d{9}$",
       "format_example": "0951234567",
       "format_regex": "^(\\d{2})(\\d{3})(\\d{4})$"
     },
     {
       "country_code": "ZW",
+      "country_name": "Zimbabwe",
```

Comment:
@prafulkoppalkar Could we take values for this from Country.res file so that there is parity and less prone to error?